### ISSUE TITLE
CO - SR results not making it to the STLT

### ISSUE THREAD

**Issue by brick-green** [CDCgov/prime-reportstream#7371]
## Problem statement

CO reported that they are not receiving results for some facilities:

Matthews VU
AFC Urgent Care in Cortez
Lake County Public Schools R-1

All three facilities say they're entering to Simple Report. We need to investigate where these results are being reported and when.

## Acceptance criteria

- [x] Verify that RS has received results for all three facilities
- [x] Verify where results were reported

## To do

- [x] Find results in metabase
- [x] Identify the outbound reports that the results were included in

**Comment by BerniXiongA6** [comment]
This is a duplicate of [ticket # 7298](https://app.zenhub.com/workspaces/onboarding--operations-6166edbd409257001e09f1b2/issues/cdcgov/prime-reportstream/7298) -- let's make sure to close out one and keep the other. Just wanted to make everyone aware. @brick-green @sliu1000

**Comment by sliu1000** [comment]
@brick-green @oslynn @BerniXiongA6 This ticket is related to 7366

**Comment by sliu1000** [comment]
@GilmoreA6 can I close this? I think this has been resolved.

**Comment by GilmoreA6** [comment]
@sliu1000 CO is still reporting issues so I don't want to close this until we verify with them

**Comment by GilmoreA6** [comment]
Hi Nancy,

Absolutely, I'm sorry to hear you are having difficulty tracking down the files so let's put another meeting on the books. Would Tuesday 11/29 at 9AM PST (11 CST) work? 

Some more follow-up on the other two facilities, Matthews-Vu and AFC. I did find the Matthews-Vu facilities and we are sending them. The issue looks like the fact they are inputting the results under a bunch of different names. Below is a count of the number of results we have gotten from Simple Report for each unique ordering facility name.

Ordering Facility Name	Count of ordering_facility_name
AFC Urgent Care	210
AFC Urgent Care - Cortez	27
AFC Urgent Care - Grand Junction	52
Matthews Vu Medical Group	17
Matthews Vu Pulsar	61
Matthews Vu Rockrimmon	12
Matthews Vu Weber Suite 120	5
Matthews Vu Weber Suite 320	4
MVMG ABC Pediatrics	2
MVMG Pediatric Specialists	1
MVMG Powers Family Med	1
Pediatrics at Matthews Vu	31
Urgent Care at Matthews Vu Weber	21
Urgent Care at Matthews-Vu	155
Grand Total	599

All these results are mostly from 10/24/2022 onward. I see only 2 from AFC Urgent Care from 02/2022 and a few from Matthews Vu from 05/22 and 06/22. From 06/22 - 10/23/2022 I don't show any results from AFC or Matthews-Vu. Let me know if you don't see results for these facilities. I did pull down and spot check some recent files and did verify they were sent, so if you are not seeing them on your side we might have to troubleshoot with a specific message to see what happened with it.

For the three feeds pointed to your test environment:

elr-test:
This receiver will only pass data that is designated as test data (MSH-11 = "T"). This is mostly used by new senders we have just onboarded to ReportStream. We leave them in "Testing" mode until we get the green light from a state or other local public health entity that we can move them to full production.

full-elr-hl7:
This is for our FHIR pilot work. This will likely be repointed to prod and eventually take the place of the production feed once the FHIR pilot is completed (currently there is no data coming down this)

elr-secondary:
The main purpose behind a secondary feed for us is usually to catch anything that doesn't qualify for the main production feed due to the quality filters. Some states have us point the failing data to a secondary (test or staging) system so they can still receive the data that doesn't pass the quality filters but not have it cause issues in their production system. If this is not something that CO wants to do than there is no reason for this to exist (this is currently turned off and does not allow anything to pass to CO on this feed as of 11/16)

As far as ReportStream vs. Simple Report it can be confusing. We do operate independently and can effectively be treated as separate entities though we do work very closely with one another.

Thanks,

James

James Gilmore (Agile Six)
Senior HL7 Data Analyst
Pandemic-Ready Interoperability Modernization Effort (PRIME)
